.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Codecov
+on:
+  push:
+    branches: [ git_workflow ]
+jobs:
+  codecov:
```

Comment:
**nit** same comment as above, we should update name

Reply:
There is two jobs in this workflow, the 1st one is for test coverage

Reply:
Yes I get that but codecov name doesn't mean anything. We can rename it to `test` at least and use the `name` prop and we can name it as `Run tests and publish test coverage`